test/library/standard/Sort/correctness/twoArrayDistSampleSort.chpl

```diff
@@ -0,0 +1,8 @@
+use BlockDist;
+use Sort, TwoArrayDistributedSampleSort;
+
+config const N = 1000;
+var A = blockDist.createArray({1..100}, int);
+twoArrayDistributedSampleSort(A);
+writeln(isSorted(A));
```

Comment:
I believe this array would be already sorted on creation. We should populate it with some random values instead.